docs/DeveloperGuide.md

```diff
@@ -154,6 +154,33 @@ Classes used by multiple components are in the `seedu.addressbook.commons` packa
 
 This section describes some noteworthy details on how certain features are implemented.
 
+### **Review Components**
+* Added Classes into the model Component to encapsulate a Review
+
+#### **Implementation**
+<img src="images/ModelReviewClassDiagram.png" width="450" />
+
+A `Review`,
+- is stored in `uniqueReviewList` of the Model
+
+A `Review` contains the following attributes,
+1. a `Name`, which represent the name of the Stall associated with the Review
+2. a `Date`, which represent the day, month and year as specified in `DD-MM-YYYY` format
```

Comment:
If I remember correctly, the display format was previously agreed to be `DD/MM/YYYY`.

Reply:
Updated in latest commit